### DISCUSSION TITLE
IfcTriangulatedFaceSet instead of IfcPolygonalFaceSet?

### DISCUSSION THREAD

**Discussion by badranX** [IfcOpenShell/IfcOpenShell#5843] [Q&A]
Edit: Sorry, the previously described problem was not related to IfcOpenShell.

I have solved the problem. If an implementation of adding IfcTriangulatedFaceSet is needed  (with ` geometry.add_mesh_representation`), I can contribute it back.

**Comment by aothms** [comment]
> If an implementation of adding IfcTriangulatedFaceSet is needed (with  geometry.add_mesh_representation), I can contribute it back.

I think that'd be a nice addition.
